tests/unit/Dns/Txt/Requests/AddRequestTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace unit\Dns\Txt\Requests;
+
+use PHPUnit\Framework\TestCase;
+use ResellerClub\Dns\Txt\Requests\AddRequest;
+use ResellerClub\TimeToLive;
+
+class AddRequestTest extends TestCase
+{
+    public function testInstantiation()
+    {
+        $ttl = TimeToLive::defaultTtl();
+        $request = new AddRequest(
+            'mytestdomain.com',
+            '@',
+            'v=spf1 ip4:123.456.789',
```

Comment:
Might have been nice to use a valid ip4 address, but no need to change unless your OCD compels you to do so :)